content/news/XXXX-XX-XX-gsoc-2025-report-armaan-chowfin.md

```diff
@@ -0,0 +1,83 @@
+title: GSoC 2025 Work Product - Resampling Options for Mixxx
+authors: Armaan Chowfin
+status: draft
+tags: gsoc, gsoc-2025
+comments: yes
+
+## Project Description
+This GSOC project is derived from [Mixxx issue#9328](https://github.com/mixxxdj/mixxx/issues/9328)
+> Mixxx uses a linear resample when scratching. This is blazing fast, but the sound can be improved. Here Mixxx should provide more resample options. This project involves to review the already used resample libraries RubberBand and Soundtouch and compare them with other candidates. The one with the best Sound/CPU load trade of shall be selected. Make sure that it supports on the fly changing of the sample rate without artefacts. This project may also involve to contribute a missing feature to such library.
```

Comment:
not sure if it is a smart move to city this literally. Your project did not solve all this. Instead we now have a recording re-sampler and a working scratching engine draft with libsamplerate without any benchmarks.